### PR TITLE
Removes cat ears from Deltastation

### PR DESCRIPTION
**Pull request by FreeStylaLT**
## What Does This PR Do
Removes publically-accessible kitty ears.

## Why It's Good For The Game
Upholds standard established in #13096

## Images of changes
Look at these cute little things:
image

Oh no!
image

## Changelog
:cl:freestylalt
del:ears of the felines from Delta
/:cl:

### CODE REVIEW

File: _maps/map_files/Delta/delta.dmm [hunk 1]
```diff
@@ -16,10 +16,6 @@
 /obj/structure/lattice,
 /turf/space,
 /area/space/nearstation)
-"abp" = (
-/turf/space,
```
Comment (AffectedArc07) [inline _maps/map_files/Delta/delta.dmm:20]: Why the heck did we have double-stacked space.

**Review by AffectedArc07 — APPROVED** [review]
We purged other methods, why should we keep this one

**Comment by Spacemanspark** [comment]
GARBAGE DAY

**Comment by HanderBoy** [comment]
Thank you!

**Comment by meow20** [comment]
Where is the "Controversial" tag

**Review by Kyep — APPROVED** [review]

**Comment by Cocacolagua** [comment]
infiltration has failed, begin plan C

**Pull request merged**